website_menu_by_user_status/README.rst

```diff
@@ -18,8 +20,9 @@ To use this module, you need to:
 #. Go to Website > Configuration > Settings > Pages.
```

Comment:
Remove "Settings":
#. Go to Website > Configuration > Pages.

Reply:
@ddufresne It might be interesting to see the new way to make a read file of the OCA https://github.com/OCA/maintainer-tools/tree/master/template/module

Reply:
@cristinamartinrod @GillesTephaneMeyomesse what's wrong with the README? Did you actually see the rendered file or did you just check the raw file ?

Also, when I submitted this PR, it respected OCA's way of making a readme. Now it does not ?

Reply:
@simahawk ok you are right. Sorry for this issue.

Reply:
I agree with you, all experienced reviewers such as @simahawk are here to improve the community code. Don't make it a personal matter when there is a note on your pull request ;)